explanation/jaas_security_scope.rst

```diff
@@ -0,0 +1,36 @@
+JAAS: Security Scope
+========================
+The scope of JAAS' security covers multiple aspects, including:
+
+Secure Communication
+~~~~~~~~~~~~~~~~~~~~
+JAAS ensures secure communication between controllers (management nodes) and models (namespaces)
+by means of authorisation, authentication and TLS encryption to prevent unauthorised access
+and prevent data interception or tampering.
+
+Additionally, when communicating with any controller, JAAS uses token based authorisation whilst
+additionally acting as the IdP (Identity Provider) for said tokens.
+
+Access control and authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+JAAS provides an abstraction layer of access control atop of Juju. JAAS does this by backing its users
+with an IdP (Identity Provider), this particular IdP is called Candid which handles the authorisation 
+and authentication for JIMM (Juju Intelligent Model Manager, the backend to JAAS). Furthermore, JAAS is 
```

Comment:
Doesn't Candid just handle authentication? JIMM handles authorization.

Reply:
I think... candid will authenticate a user and provide a macaroon with some basic caveats which I guess are a form of authorisation? We don't have centralised authz but I'll reword it as candid = authn, jimm = authz as it's more of an implementation detail probably